applicant-frontend/src/Providers/auth.js

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
I don't think that it's really useful to put these functions into a new file. I think that you can add them directly into the components. It's as you want.